src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpInstantiateArgumentExceptionsCorrectly.Fixer.cs

```diff
@@ -13,5 +20,77 @@ namespace Microsoft.NetCore.CSharp.Analyzers.Runtime
     [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
     public sealed class CSharpInstantiateArgumentExceptionsCorrectlyFixer : InstantiateArgumentExceptionsCorrectlyFixer
     {
+        protected override void PopulateCodeFix(CodeFixContext context, Diagnostic diagnostic, string paramPositionString, SyntaxNode node)
+        {
+            if (node is ObjectCreationExpressionSyntax creation)
+            {
+                int paramPosition = int.Parse(paramPositionString, CultureInfo.InvariantCulture);
+                CodeAction? codeAction = null;
+                if (creation.ArgumentList.Arguments.Count == 1)
+                { // Add null message
```

Comment:
Nit: we generally avoid a brace and comment on the same line. Suggest moving the comment to a new line.